ambry-api/src/test/java/com.github.ambry/rest/MockBlobStorageService.java

```diff
@@ -108,6 +108,13 @@ public void handlePost(RestRequest restRequest, RestResponseChannel restResponse
     }
   }
 
+  /* Make sure PUT is not supported  */
```

Comment:
minor: The java doc here could be similar to what is in `AdminBlobStorageService`